source/cli/tests/test_version.py

```diff
@@ -0,0 +1,5 @@
+from instance_scheduler_cli import __version__
+
+
+def test_version_correctly_picked_up_from_toml():
+    assert __version__ == "1.5.0"
```

Comment:
Will this break when the version changes?

Reply:
yes, it will. we can remove this test entirely when that comes, but I figured a test breaking in that scenario was the lesser evil compared to not being certain that my changes to the "--version" command worked correctly.